app/models/user_session.rb

```diff
@@ -2,8 +2,10 @@ class UserSession < Authlogic::Session::Base
   allow_http_basic_auth false
   consecutive_failed_logins_limit 10
   failed_login_ban_for 30.minutes
-  generalize_credentials_error_messages 'Login failed due to invalid ' \
-                                        'username and/or password.'
+  generalize_credentials_error_messages 'These credentials are not recognized as valid' \
```

Comment:
@peggles2 , this is getting hard to read. Let's clean this up with a constant (which we should freeze):
```ruby
# frozen_string_literal: true

class UserSession < Authlogic::Session::Base
  INVALID_LOGIN_MESSAGE = <<~MESSAGE.chomp
    These credentials are not recognized as valid for accessing Search.gov.
    Please contact search@support.digitalgov.gov if you believe this is in error.
  MESSAGE

  ...
  generalize_credentials_error_messages INVALID_LOGIN_MESSAGE
```